index.html

```diff
@@ -131,7 +131,7 @@ <h3 class="project-title">Tonic</h3>
         </div> <!-- .project -->
 
         <div class="project">
-          <img src="./images/works-section/project4.png" alt="Fourth project image">
+          <img src="./images/works-section/project4.svg" alt="Fourth project image" class="project-image right-side">
```

Comment:
@tufoinnkuo10 those margins are added from live-server (which I wasn't conscious about it) just in case, I put an !important to margin: 0; on CSS. Thank you for letting me know!

Reply:
Good job following the advice of the previous reviewer